bigquery-credentials.json.ctmpl

```diff
@@ -0,0 +1,15 @@
+{{- define "KEY" -}}
```

Comment:
docker hub requires Dockerfiles to be in the top level. Though I am unsure about using automated builds -- for now lets keep this, but if we decided to not do automated builds we can move the Dockerfiles.